source/_integrations/adguard.markdown

```diff
@@ -109,3 +109,11 @@ Refresh all filter subscriptions in AdGuard Home.
 
 By default, `force` is set to `false`. Forcing an update bypasses AdGuard Home's
 throttling logic, so use with care.
+
+## Encyption
+
+AdGuard supports DNS-over-HTTPS and DNS-over-TLS. 
+
+<div class="note">
```

Comment:
The integration does not handle this. Did you mean to open a change for the add-on?